test/unit/keycloak.test.js

```diff
@@ -264,6 +266,37 @@ describe('keycloak', () => {
       expect(updateKeycloakConfigMock.callCount).to.equal(0);
       keycloak.__ResetDependency__('_keycloakConfig');
     });
+
+    it('should call keycloak.updateToken() when action has 401 or 403 error', () => {
```

Comment:
Stylistic nit: these might be "cleaner" as two separate test cases.

Reply:
Wouldn't hurt to split this out. 👍🏼